stdlib/public/Concurrency/PartialAsyncTask.swift

```diff
@@ -303,15 +303,26 @@ public struct ExecutorJob: Sendable, ~Copyable {
   }
 
   public var priority: JobPriority {
-    let raw: UInt8
-    if #available(SwiftStdlib 6.2, *) {
-      raw = _jobGetPriority(self.context)
-    } else {
-      // We are building the new version of the code, so we should never
-      // get here.
-      Builtin.unreachable()
+    get {
+      let raw: UInt8
+      if #available(SwiftStdlib 6.2, *) {
+        raw = _jobGetPriority(self.context)
+      } else {
+        // We are building the new version of the code, so we should never
+        // get here.
+        Builtin.unreachable()
+      }
+      return JobPriority(rawValue: raw)
+    }
+    set {
```

Comment:
Hmmm are we really ok with making that settable by end users...? It would not cause priority escalation of child tasks if someone just randomly sets these like that etc Seems problematic?

Reply:
Chatted offline that it's for testing, so maybe we can SPI it